lib/omnibus/project.rb

```diff
@@ -1615,6 +1618,14 @@ def shasum
       end
     end
 
+    def disable_healthcheck(val)
+      if val.nil?
+        @disable_healthcheck
+      else
+        @disable_healthcheck = val
+      end
+    end
```

Comment:
The DSL "purist" in me would probably prefer to have this method not care about returning a value so that's easier to implement it without requiring an argument (that would be more natural to me), but feel free to ignore:

```suggestion
    def disable_healthcheck
      @disable_healthcheck = true
    end
```

Apart from that, renaming this to `skip_healthcheck` would be more consistent with e.g. https://github.com/DataDog/omnibus-ruby/blob/939e1cc40f079de839d4198cd39e82a55cc4e6e1/lib/omnibus/software.rb#L470C1-L471C1

Reply:
Renamed the function, and keeping the parameter for consistency with other similar methods. Although for the record I agree with you that we shouldn't have to care about returning the value